layouts/game/list.html

```diff
@@ -1,61 +1,198 @@
+                  <span hidden>{{ .Params.title }} {{ $rating.name }}</span>
+               </td>
+               <td class="game-title" data-title="{{ .Params.title }}">
+                  <a href="{{ .Permalink }}">{{ .Params.title }}</a>
+               </td>
+               <td class="compatibility" data-compatibility="{{ $rating.key }}">
+                  <div class="is-square" style="background-color: {{ $rating.color }}"></div>
+                  <span>{{ $rating.name }}</span>
+               </td>
+               <td class="date-tested" data-timestamp="{{ .Params.testcase_date }}">
+                  {{ if .Params.testcase_date }}
+                  <span>{{ dateFormat "January 2, 2006" .Params.testcase_date }}</span>
+                  {{ else }}
+                  <span></span>
+                  {{ end }}
+               </td>
             </tr>
             {{ end }}
-        </tbody>
-    </table>
+         </tbody>
+      </table>
+      <ul class="pagination paginationBottom is-pulled-left"></ul>
+   </div>
 </div>
 {{ end }}
 
 {{ define "scripts" }}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js" 
+        integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/list.js/2.3.1/list.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.9.1/chart.min.js"></script>
+
+<script type="text/javascript">
+    $(function() {
+        function getLastSorted(list){
+            var currentSorted = $(list.listContainer).find(".asc, .desc").attr("data-sort");
+            if ('undefined' !== typeof currentSorted)
+                list.lastSorted = currentSorted;
+            return list.lastSorted;
+        }
+ 
+        function filterTable(index, name) {
+            $("#search-box").val(name);
+            list.search(index, ['compatibility']);
+        }
+ 
+        function clearSearch() {
+            $("#search-box").val("");
+            list.search();
+        }
+ 
+        var options = {
+            valueNames: [
+                'listing-metadata',
+                { name: 'game-title', attr: 'data-title' },
+                { name: 'compatibility', attr: 'data-compatibility' },
+                { name: 'date-tested', attr: 'data-timestamp' }
+            ],
+            searchColumns: [ 'listing-metadata' ],
+            pagination: [
+                { paginationClass: "paginationTop" },
+                { paginationClass: "paginationBottom" }
+            ],
+            indexAsync: true,
+            page: 30
+        };
+ 
+        var list = new List('game-listing', options);
+ 
+        list.sort("title", {order: "asc"});
+ 
+        list.on("updated", function(list){
+            var $list = $(list.list);
+            var $items = $list.children();
+            $items.removeClass("key");
+            if ('title' == getLastSorted(list)){
```

Comment:
```suggestion
            if ('title' === getLastSorted(list)){
```

In JavaScript, it's recommended to use full equals (`===`) to avoid implicit type conversion (aka "type juggling" in some other languages).